spdx/parsers/tagvaluebuilders.py

```diff
@@ -1248,13 +1248,13 @@ def add_file_dep(self, doc, value):
 
     def set_file_atrificat_of_project(self, doc, symbol, value):
         """
-        Set a file name, uri or home artificat.
```

Comment:
The artificat strikes again!
(Sorry for the nonsensical comment, but I find this typo hilarious)